### PR TITLE
docs(monitoring/datadog): First pass at Datadog setup guide

### PR DESCRIPTION
**Pull request by benjaminws**
Here is a first pass at some words on the setting up Datadog with spinnaker-monitoring.

### CODE REVIEW

**Review by lwander — COMMENTED** [review]
Thanks for the writeup! I am not very familiar with the monitoring side of things, could @ewiseblatt have a look?

File: setup/monitoring/datadog.md [hunk 1]
```diff
@@ -6,3 +6,95 @@ sidebar:
...
+It is not required that you run a Datadog agent for this setup to work properly,
+but it is recommended to provide greater visibility into your spinnaker installation.
+If you do chose to run the agent, you can find more information on the the Datadog website here: [Agent - Datadog Docs](https://docs.datadoghq.com/agent/).
+
+Additionally, you can install the agent with the `spinnaker-third-party` scripts as outlined below.
+This document assumes a debian installation.
+
+## Install the Spinnaker Monitoring Daemon
+
+To get started need to make sure the spinnaker-monitoring-daemon is installed on your machines.
```
Comment (ghost) [inline setup/monitoring/datadog.md]: ...get started **you'll** need to make...

**Review by ghost — APPROVED** [review]
One minor language nit but otherwise this lgtm!

File: setup/monitoring/datadog.md (hunk 1 above)
Comment (benjaminws) [reply setup/monitoring/datadog.md]: Fixed :)

**Comment by benjaminws** [comment]
Think we could go ahead and merge this? I'm comfortable this as a first-pass.

**Pull request merged**